app/src/main/java/zapsolutions/zap/fragments/SettingsFragment.java

```diff
@@ -313,11 +298,7 @@ public void onClick(DialogInterface dialog, int whichButton) {
 
     @Override
     public void guardianDialogConfirmed(String DialogName) {
```

Comment:
Should we remove `UserGuardianInterface` and `UserGuardian` from the class, since its not used anymore?

Reply:
Yes, thats not used anymore. I will remove it.